linux/keyman-config/keyman_config/install_kmp.py

```diff
@@ -234,6 +239,13 @@ def install_kmp_user(inputfile, online=False):
 			elif ftype == KMFileTypes.KM_SOURCE:
 				#TODO for the moment just leave it for ibus-kmfl to ignore if it doesn't load
 				pass
+			elif ftype == KMFileTypes.KM_KMX:
+				# Sanitize keyboard filename if not lower case
+				kmx_id = re.sub('.kmx', '', f['name'])
+				for index, kb in enumerate(keyboards):
+					if kmx_id.lower() == kb['id'] and kmx_id != kb['id']:
```

Comment:
Are there any cases we have to handle where the keyboard id kb['id'] isn't lower case?

Reply:
The keyboard ID should always be lower case